src/main/java/lt/vu/web/api/v1/controller/user/GetCurrentUserInfoController.java

```diff
@@ -0,0 +1,46 @@
+package lt.vu.web.api.v1.controller.user;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import lt.vu.infrastructure.security.Authorized;
+import lt.vu.web.api.v1.controller.security.CurrentUserAwareController;
+import lt.vu.web.api.v1.dto.user.GetUserDTO;
+import lt.vu.web.api.v1.exception.ExceptionDTO;
+
+import javax.enterprise.context.RequestScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/currentUser")
```

Comment:
Best practices tell us it's best to use `kebab-case` for URLs. Also, I'd just keep it as `/user`.